### ISSUE TITLE
Sync command only works in container's directory

### ISSUE THREAD

**Issue by mpcarolin** [simpleviewinc/keg-cli#13]
## Context
I am developing for tap-events-force, and I frequently need to sync with keg-core to develop on both simultaneously

> My `keg-cli` version: 1.4.1

## Problem
Running `keg <tap_name> sync core` fails if I run it in a keg-* directory other than the tap's. It seems to me that the command should not care where it is run.

Error encountered:

```bash
 Invalid dependency path. The dependency path can not be the same as the context path.
Context Path: /Users/michael.carolin/keg/keg-core
Dependency Path: /Users/michael.carolin/keg/keg-core
```

## Steps to reproduce
1. `keg d clean`
2. `keg core build`
3. `keg <tap_name> start`
4.  cd <path_to_core>
5. `keg <tap_name> sync core`
6. Observe error.
7. cd <path_to_tap>
8. `keg <tap_name> sync core
9. Observe no errors and sync working

**Comment by lancetipton** [comment]
This should be resolved with #22

**Comment by mpcarolin** [comment]
Yep, it's fixed!
